### PR TITLE
3947: HBX Admin is shifted from Pre-Prod to Prod when selecting a consumer name in the Families tab (HBX Portal)

### PR DESCRIPTION
**Pull request by 202chaz**
|Field|Value|
|-----|-----|
| Redmine Ticket Number | [3947](https://devops.dchbx.org/redmine/issues/3947) |
| App-Dev Road Map # | n/a |
| Start Date | 11/13/2015 |
| Due Date | n/a  |
| App-Dev Priority | n/a |
| Development Story Points | n/a |
| Peer Review Story Points | n/a |
| Ticket Author | Jack Weiner |

### CODE REVIEW

File: script/remove_prod_urls_from_bookmarks.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+p = Person.where(is_active:true)
+
+p.each do |c|
+  if c.present?
+    if c.consumer_role.present?
+      if c.consumer_role.bookmark_url.present?
+        link = c.consumer_role.bookmark_url.gsub(/enroll.dchealthlink.com/, 'enroll-preprod.dchbx.org')
```
Comment (Kevin-Wei) [inline script/remove_prod_urls_from_bookmarks.rb]: @ChazHaskins can we change this to relative?

**Comment by 202chaz** [comment]
@Kevin-Wei Updated to use relative urls

**Comment by davidplappert** [comment]
OLD REQUEST

**Comment by Kevin-Wei** [comment]
Another old data sync issue that was resolved via data script or refresh

**Pull request closed without merging**